src/suites/cts/validation/createBindGroupLayout.spec.ts

```diff
@@ -3,10 +3,14 @@ createBindGroupLayout validation tests.
 `;
 
 import { C, TestGroup, poptions } from '../../../framework/index.js';
-import { bindingTypeInfo, bindingTypes } from '../format_info.js';
+import { ParamSpec } from '../../../framework/params/index.js';
+import { bindingTypeInfo, bindingTypes, shaderStages } from '../format_info.js';
```

Comment:
Maybe not for this PR, but it would be nice if these constant exports from format_info had different casing. Multiple times in this review I was looking for local definitions of `bindingTypes` or `shaderStages` . Maybe `kBindingTypeInfo`, `kBindingTypes`, and `kShaderStages` would be more clear.